ci-operator/jobs/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openstack-k8s-operators/glance-operator:
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
This should remains as `false`
Is the script that is making this change? In my environment, it doesn't provide this line change.
Are you using `make jobs` to fix this file?

Reply:
i ran make update and that made this change.